verify-ui/.env

```
@@ -1,12 +1,13 @@
 DEFAULT_TITLE=Inji Verify
 DEFAULT_THEME=default_theme
 DEFAULT_LANG=en
+DEFAULT_FONT_FAMILY=Inter,sans-serif
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEFAULT_FONT_FAMILY key should go before the DEFAULT_LANG key

(UnorderedKey)